src/utils/rest-api.js

```diff
@@ -2205,6 +2205,31 @@ export function getSensiDefaultResultsThreshold() {
     });
 }
 
+export function fetchMapEquipments(
+    studyUuid,
+    currentNodeUuid,
+    substationsIds,
+    inUpstreamBuiltParentNode
+) {
+    console.info(
+        `Fetching map equipments data of study '${studyUuid}' and node '${currentNodeUuid}'...`
+    );
+    let urlSearchParams = new URLSearchParams();
+    if (inUpstreamBuiltParentNode !== undefined) {
+        urlSearchParams.append(
+            'inUpstreamBuiltParentNode',
+            inUpstreamBuiltParentNode
+        );
+    }
+
+    const fetchEquipmentsUrl =
+        getStudyUrlWithNodeUuid(studyUuid, currentNodeUuid) +
+        '/network-map/map-equipments-data?' +
```

Comment:
If substationsIds is empty you will have an URL finiching with `?` which is not really clean.
Something like this should do the trick :
```
const paramsList = substationsIds && substationsIds.length > 0
        ? '?' + getQueryParamsList(substationsIds, 'substationId')
        : '';
```

Reply:
Agreed, done